src/Data/Nat/Properties.agda

```diff
@@ -1185,6 +1190,46 @@ m⊔n≤m+n m n with ⊔-sel m n
 ... | inj₁ m⊔n≡m rewrite m⊔n≡m = m≤m+n m n
 ... | inj₂ m⊔n≡n rewrite m⊔n≡n = m≤n+m n m
 
+------------------------------------------------------------------------
+-- Other properties of _⊔_ and _*_
+
+*-distribˡ-⊔ : _*_ DistributesOverˡ _⊔_
+*-distribˡ-⊔ m zero o = begin-equality
+  m * (zero ⊔ o)                                        ≡⟨⟩
+  m * o                                                 ≡⟨⟩
+  zero ⊔ m * o                                          ≡˘⟨ cong (_⊔ m * o) (*-zeroʳ m) ⟩
+  m * zero ⊔ m * o                                      ∎
+*-distribˡ-⊔ m (suc n) zero = begin-equality
+  m * (suc n ⊔ zero)                                    ≡⟨⟩
+  m * suc n                                             ≡˘⟨ ⊔-identityʳ (m * suc n) ⟩
+  m * suc n ⊔ zero                                      ≡˘⟨ cong (m * suc n ⊔_) (*-zeroʳ m) ⟩
+  m * suc n ⊔ m * zero                                  ∎
+*-distribˡ-⊔ m (suc n) (suc o) = begin-equality
+  m * (suc n ⊔ suc o)                                   ≡⟨⟩
+  m * suc (n ⊔ o)                                       ≡⟨ *-suc m (n ⊔ o) ⟩
+  m + m * (n ⊔ o)                                       ≡⟨ cong (m +_) (*-distribˡ-⊔ m n o) ⟩
+  m + (m * n ⊔ m * o)                                   ≡⟨ +-distribˡ-⊔ m (m * n) (m * o) ⟩
+  (m + m * n) ⊔ (m + m * o)                             ≡˘⟨ cong₂ _⊔_ (*-suc m n) (*-suc m o) ⟩
+  (m * suc n) ⊔ (m * suc o)                             ∎
+
+*-distribʳ-⊔ : _*_ DistributesOverʳ _⊔_
+*-distribʳ-⊔ m zero o = refl
```

Comment:
Much easier to use `comm+distrˡ⇒distrʳ` from `Algebra.Consequences.Propositional` for the right versions of distributivity.

https://github.com/agda/agda-stdlib/blob/eb338fd27861f8c9db1359fda6e0ad24df555ae0/src/Algebra/Consequences/Propositional.agda#L76